backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/cgen/CBridgeGen.kt

```diff
@@ -306,7 +310,11 @@ internal fun KotlinStubs.generateObjCCall(
         receiver: ObjCCallReceiver,
         arguments: List<IrExpression?>
 ) = builder.irBlock {
-    val callBuilder = KotlinToCCallBuilder(builder, this@generateObjCCall, isObjCMethod = true)
+    val exceptionMode = ForeignExceptionMode.byValue(
+            method.module.konanLibrary?.manifestProperties?.getProperty(ForeignExceptionMode.manifestKey)
```

Comment:
A compiler's IR "feature" might affect this code in a harmful way.

There are "fake overrides".  When you call a method that is defined in a superclass, compiler IR encodes it as a call to a "fake override" in a subclass. E.g.
```kotlin
class NSObjectImpl : NSObject()

fun main() {
    NSObjectImpl().description()
}
```
In this case the compiler synthesizes "fake override" method `description` in `NSObjectImpl`, and IR contains `IrCall` to this `NSObjectImpl.description`. So the code here may accidentally inspect this method and query the manifest for the library containing this method, which is not even an interop library.

This is usually addressed with `method.resolveFakeOverride(allowAbstract = true)`.